### PR TITLE
Add ENV_VARS to bok-choy build flow.

### PR DESCRIPTION
**Pull request by benpatterson**
The subset job has been updated to include the ENV_VARS parameter.

That parameter is a comma-delimited set of environment variables that will be set using the [EnvInject Plugin](https://wiki.jenkins-ci.org/display/JENKINS/EnvInject+Plugin). The parameter string is parsed and individual environment variables are set in the environment by using the groovy utility of that plugin.

Using this approach, the parent job (build flow job) can receive those parameters to pass down to the child jobs. 

This means we can set various important environment variables as needed for running bok-choy. The best example is setting the SELENIUM_BROWSER variable to something else, or likewise passing the new accessibility-related settings.

@clytwynec @jzoldak could you review? 

An example of this is [here for your inspection](https://build.testeng.edx.org/view/All/job/edx-platform-bok-choy-bjp/21/). Notice that this is testing in chrome, which is analogous to a hard-coded set of jobs [here](https://build.testeng.edx.org/view/All/job/edx-platform-on-chrome).

CC @raeeschachar

### CODE REVIEW

File: jenkins/flow/master/edx-platform-bok-choy-master.groovy [hunk 1]
```diff
@@ -7,7 +7,7 @@ def jenkinsUrl = build.environment.get("JENKINS_URL")
 def jobUrl = jenkinsUrl + build.url
 
 try{
-  def statusJobParams = [
+  def statusJobparams = [
```
Comment (clytwynec) [inline jenkins/flow/master/edx-platform-bok-choy-master.groovy]: Why is this change needed?

Comment (benpatterson) [inline jenkins/flow/master/edx-platform-bok-choy-master.groovy]: Och. That's a typo as I was working on it. Will fix.

**Comment by benpatterson** [comment]
@clytwynec I fixed the typo. Ready for 2nd pass.

**Comment by jzoldak** [comment]
nice @benpatterson!

:+1:

**Comment by clytwynec** [comment]
:+1:
